examples/pytorch/dtgrnn/dcrnn.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+import scipy.sparse as sparse
+import torch
+import torch.nn as nn
+import dgl
+from dgl.base import DGLError
+import dgl.function as fn
+
+
+class DiffConv(nn.Module):
+    '''DiffConv is the implementation of diffusion convolution from paper DCRNN
+    It will compute multiple diffusion matrix and perform multiple diffusion conv on it,
+    this layer can be used for traffic prediction, pedamic model.
+    Parameter
+    ==========
+    in_feats : int
+        number of input feature
+
+    out_feats : int
+        number of output feature
+
+    k : int
+        number of diffusion steps
+
+    dir : str [both/in/out]
+        direction of diffusion convolution
+        From paper default both direction
+    '''
+    in_graph_list = []
```

Comment:
Don't make these as static. Use diffussed_graph as input in the forward function.